### ISSUE TITLE
One keystroke is printing two characters, during install.

### ISSUE THREAD

**Issue by mrillusi0n** [m4heshd/whatsapp-desktop-dark#32]
My keystrokes: #ff847c
On screen: ##884477cc

But the input was valid.

**Comment by exadeci** [comment]
Same issue with 0.3.4678 on osx

I tried with Iterm2 and bash (I use fish) but the issue persists.

[edit] Also on Mojave

**Comment by mrillusi0n** [comment]
If it helps to narrow down the issue, I faced this on macOS (Mojave) too. Additionally, used fish terminal, as @exadeci.

**Comment by KeronCyst** [comment]
Same on Win10!

**Comment by m4heshd** [comment]
It's a known issue on Node.js which i included [right here](https://github.com/m4heshd/whatsapp-desktop-dark#known-issues) in the documentation. Take a look at https://github.com/nodejs/node/issues/25875.

**Comment by m4heshd** [comment]
Surprisingly I've managed to fix this right now. Please check the binary downloads and running from the source.

**Comment by mrillusi0n** [comment]
Sorry for not going through the documentation properly. Closing this issue, since it's already documented.

**Comment by m4heshd** [comment]
@mrillusi0n, No problem at all. Can i label this as "solved"? Need some confirmation before i proceed.

**Comment by exadeci** [comment]
@m4heshd Could you share with us a compiled version? I've installed node `npm install` failed, then tried with nvm and the lts node but it failed again with the same dependency install issue:
```
Downloading ResourceHacker by Angus Johnson...
_http_client.js:107
  var expectedProtocol = defaultAgent.protocol;
                                      ^

TypeError: Cannot read property 'protocol' of undefined
    at new ClientRequest (_http_client.js:107:39)
```

**Comment by m4heshd** [comment]
@exadeci, It's already updated in the latest release.

**Comment by mrillusi0n** [comment]
@m4heshd It's technically not solved, but I thought the issue would not make sense when it's already mentioned under the docs.

**Comment by m4heshd** [comment]
@mrillusi0n did you download the release again? Because I've tested on multiple distributions including macOS Mojave and the inputs work fine now. I've got some other people to test too. All reported to be working as it was suppose to work.

**Comment by exadeci** [comment]
Yeah it works now, might be better next time to name it with a different revision like 0.3.467**9** because the release page doesn't display if a file has been changed.

**Comment by m4heshd** [comment]
@exadeci I can't do that since I version the dark mode releases parallel to WhatsApp desktop releases. It's in the documentation. It's always good practice to keep the source versioning aligned with the target when you're releasing tweaks/plugins for another application to avoid confusion. Also if i did that, it would show all the other users that a new version is available since the script now support auto update checks, which would've confused people more. Ironically I updated this release just before WhatsApp released a bugfix patch tagged `0.3.4679`. I still stand by my decision because it's a good thing so i can save that minor version difference to push useful updates such as broken styles since the patch is still super early to debug.

**Comment by mrillusi0n** [comment]
> @mrillusi0n did you download the release again? Because I've tested on multiple distributions including macOS Mojave and the inputs work fine now. I've got some other people to test too. All reported to be working as it was suppose to work.

**TL;DR** I did not face the issue in the newest binary.

Yes, I did download, and the problem does not exist. Another observation I made was that the problem did not occur when I did `node .`, but I did face the same issue when I tried `npm run dist-mac`, although, I've not tried building it after the newest commit.
